bin/docs/debugger.txt

```diff
@@ -1,43 +1,59 @@
-disassembly view:
-     
-    ctrl+g          goto
-    ctrl+e          edit breakpoint
-    ctrl+d          enable/disable breakpoint
-    ctrl+b          add breakpoint
-    right           follow branch/position memory view to accessed address
-    left            go back one branch level/goto pc
-    up              move cursor up one line
-    down            move cursor down one line
-    page up         move visible area up one page
-    page down       move visible area down one page
-    f10             step over
-    f11             step into
-    tab             toggle display symbols
-    left click      select line/toggle breakpoint if line is already highlighted
-    right click     open context menu
-     
-memory view:
-     
-    ctrl+g          goto
-    ctrl+b          add breakpoint
-    left            move cursor back one byte/nibble
-    right           move cursor ahead one byte/nibble
-    up              move cursor up one line
-    down            move cursor down one line
-    page up         move cursor up one page
-    page down       move cursor down one page
-    0-9,A-F         overwrite hex nibble
-    any             overwrite ansi byte
-    left click      select byte/nibble
-    right click     open context menu
-    ctrl+wheel      zoom memory view
-    esc             return to previous goto address
-    ctrl+v          paste a hex string into memory
+†---------------------------------------------------------------------------------†
+   
+Disassembly view:
+   
+†---------------------†
```

Comment:
Visual separation
Begin and end of file get longer dashes, titles get smaller dashes.
If you have a better idea to not make it a blob or not have decent formatting, i am all ears.
The way the debugger.txt is used , it's actually splitting the strings from it's own side, so yeah it all looks a bit wrong due to me trying to workaround it.

Master:
![image](https://user-images.githubusercontent.com/24227051/117541787-c3c1ec80-b015-11eb-8cef-931aa0d11bf8.png)

PR:
![image](https://user-images.githubusercontent.com/24227051/117541765-b4db3a00-b015-11eb-9b0d-cd765ccc0836.png)

Reply:
While the actual text file looks like this:
![image](https://user-images.githubusercontent.com/24227051/117541950-bbb67c80-b016-11eb-9754-a80ee50e20af.png)

Reply:
I've remade it into this:

![image](https://user-images.githubusercontent.com/24227051/117542355-8e6ace00-b018-11eb-8813-5c0004a1c8c6.png)

Reply:
Ok that's pretty effin bad, do you think you can get a monospace font to be used in the text viewer so the help looks somewhat ok? Because this just looks bad because of the variable spacing

Reply:
@RedDevilus you can, see: https://docs.wxwidgets.org/3.0/classwx_text_ctrl.html
`virtual bool wxTextCtrl::SetDefaultStyle 	( 	const wxTextAttr &  	style	)`